specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -316,6 +316,151 @@
+        "tags": [
+          "vNetPeering"
+        ],
+        "operationId": "vNetPeering_Delete",
+        "description": "Deletes the workspace vNetPeering.",
+        "x-ms-examples": {
+          "Delete a workspace": {
+            "$ref": "./examples/WorkspaceVirtualNetworkPeeringDelete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PeeringNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted - Returns this status until the asynchronous operation has completed."
+          },
+          "204": {
+            "description": "NoContent"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "put": {
+        "tags": [
+          "vNetPeering"
+        ],
+        "operationId": "vNetPeering_CreateOrUpdate",
+        "description": "Creates vNet Peering for workspace.",
+        "x-ms-examples": {
+          "Create or update workspace": {
+            "$ref": "./examples/WorkspaceVirtualNetworkPeeringCreate.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "VirtualNetworkPeeringParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkPeering"
+            },
+            "description": "Parameters supplied to the create workspace vNet Peering."
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PeeringNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Created - Returns information about the workspace vNet peering, including provisioning status.",
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkPeering"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
if it's a long running operation, should there be other responses code other than 200?

Reply:
no it is not long running operation, I missed to remove it